src/cli.js

```diff
@@ -10,7 +10,7 @@ Options:
   -h, --help            Bring help.
   --version             Show version.
   -v, --verbose         Enable verbose mode.
-  -d, --dest=<dir>      Documentation folder [default: sassdoc].
```

Comment:
See commit message and Slack discussion; it was overriding any config-defined `dest` even if not explicitely given via CLI.